tests.ml

```diff
@@ -285,20 +285,20 @@ c:
  clear r
 "
 
-let test_branch_pruned =
-"function main ()
-version anon_1
- mut x = 9
+let test_branch_pruned = " mut x = 9
```

Comment:
Nitpick: Can the string be moved to the next line? For a moment, I almost missed the `mut x = 9`, and was confused at the `mut x = &x` below.